### PR TITLE
commit changes

### PR DESCRIPTION
**Pull request by meetpatel2512**
i have complete task of PR

### CODE REVIEW

File: pr.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PR</title>
+</head>
+
+<body>
+    <h1>Hello</h1>
+    <p>Lorem ipsum dolor sit amet.</p>
```
Comment (the-special-character) [inline pr.html]: please write appropriate text

File: pr.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PR</title>
```
Comment (the-special-character) [inline pr.html]: give proper title

**Review by the-special-character — CHANGES_REQUESTED** [review]
please make changes as per request

File: pr.html (hunk 1 above)
Comment (meetpatel2512) [reply pr.html]: done changes according to request

File: pr.html (hunk 2 above)
Comment (meetpatel2512) [reply pr.html]: done change

**Review by the-special-character — APPROVED** [review]
good work

**Pull request merged**